### ISSUE TITLE
Windows 11: Smart App Control blocks the installation

### ISSUE THREAD

**Issue by KHerzberg** [texstudio-org/texstudio#3689]
### Environment
- TeXstudio: 4.8.1
- OS: Windows 11

### Expected behavior
The installation program is started by double-clicking on texstudio-4.8.1-win-qt6.exe.

### Actual behavior
See Screenshot.
![SmartApp](https://github.com/texstudio-org/texstudio/assets/76074983/14d47e68-f090-4c1d-a884-340c3c5c599a)

### How to reproduce
Activate Smart App Control.
![image](https://github.com/texstudio-org/texstudio/assets/76074983/2055fe20-75fe-49d3-9f86-ddfd05aab741)

Smart App Control adds significant protection from new and emerging threats by blocking apps that are malicious or untrusted.
https://support.microsoft.com/en-us/topic/what-is-smart-app-control-285ea03d-fa88-4d56-882e-6698afdb7003#bkmk_unknown

**Comment by sunderme** [comment]
this is rather unlikely as a certificate costs >$150 per year.

**Comment by timhoffm** [comment]
@sunderme I'm not active anymore with the project, so none of my business. But if you want to generate revenue for a certificate, probably the easiest way is to activate github sponsors. Of course it's administrative overhead and you also have to care for taxes. So it's a decision whether you are willing to do the extra effort.

**Comment by KHerzberg** [comment]
I just happened to come across a solution that may have to be downgraded to a workaround. The "Winget" tool was introduced as standard in Windows 11. The Linux pandent for this in Debian distributions is known as "sudo apt" or "sudo apt-get".

To install Textstudio, simply enter the command "winget install texstudio" in an admin powershell (not cmd). That's it. The latest version is downloaded from Git and installed. In my case, only an update is carried out, as I managed to install an older version of TexStudio.
![image](https://github.com/texstudio-org/texstudio/assets/76074983/276875e1-dbc5-4d55-892b-24122b8f4409)

For updates, the command "winget upgrade texstudio" or, if you want to update all programs on the computer, the command "winget upgrade --all" is used.
![image](https://github.com/texstudio-org/texstudio/assets/76074983/dc1e2da3-a7e8-4bca-b6e3-412a73c51848)


The big question is how long Microsoft will allow this workaround. It is actually nothing more than the .exe provided.
